getsentry/sentry
# proto(hybrid-cloud): Cross-silo model handling with RPCs

This PR creates a dummy endpoint with simple model changes and a proposed RPC layer for making model changes across silos.

To test:
- Spin up two instances of sentry using [these instructions](https://github.com/getsentry/sentry/blob/master/src/sentry/silo/README.md)
- Using the Sentry Silo Mode environment variables, a new service called `silo-rpc` should be spun up
<img width="891" alt="image" src="https://user-images.githubusercontent.com/35509934/199101249-d882b92e-0003-45f9-a8aa-62a1552cf323.png">
- Now using the `/api/0/silo-debug/(user | organization)/` endpoint with a json payload shaped like:

```json
{
    "id":  1,
    "name": "new user.username or org.name"
}
```
- It should perform the change regardless of the fact that the model lives in the other silo! See demo:


https://user-images.githubusercontent.com/35509934/199100433-57d380d4-71b2-467c-94da-72679b4230e9.mov
